run_specific_test.sh

```diff
@@ -5,4 +5,5 @@ export CARAVEL_CONFIG=tests.caravel_test_config
 set -e
 caravel/bin/caravel version -v
 export SOLO_TEST=1
-nosetests tests.core_tests:CoreTests.test_public_user_dashboard_access
+#nosetests tests.core_tests:CoreTests.test_slice_endpoint
```

Comment:
Oh this is just a little helper file to run a specific tests without re-running all the prep work (loading examples, init, ...). typically would just point it to whatever I'm iterating on at the time. I'll change it to receive an argument in the future.